scarb/src/compiler/compilers/starknet_contract.rs

```diff
@@ -216,6 +223,14 @@ impl Compiler for StarknetContractCompiler {
         let compiler_config = build_compiler_config(&unit, ws);
 
         let main_crate_ids = collect_main_crate_ids(&unit, db);
+        // main crate ids to definicja ze scarb.tomla glownego?
+        eprintln!("unit = {:#?}", unit);
+        eprintln!("unit.main_component() = {:#?}", unit.main_component());
+        eprintln!(
+            "unit.main_component().cairo_package_name() = {:#?}",
+            unit.main_component().cairo_package_name()
+        );
+        eprintln!("main_crate_ids = {:#?}", main_crate_ids);
```

Comment:
```suggestion
```

btw https://doc.rust-lang.org/stable/std/macro.dbg.html

Reply:
woops, was almost sure that i didn't push that, my bad 😅